server/src/main/scala/uk/gov/homeoffice/drt/service/staffing/ShiftsService.scala

```diff
@@ -48,13 +48,14 @@ case class ShiftsServiceImpl(staffShiftsDao: StaffShiftsDao)(implicit val ec: Ex
     Future.sequence(shifts.map(staffShiftsDao.insertOrUpdate)).map(_.sum)
   }
 
-  override def deleteShift(port: String, terminal: String, shiftName: String): Future[Int] = staffShiftsDao.deleteStaffShift(port, terminal, shiftName)
+  override def deleteShift(previousShift: Shift): Future[Shift] = staffShiftsDao.deleteStaffShift(previousShift.port,
```

Comment:
why is the shift argument called 'previousShift' here? Is it not just the shift you want to delete, is `shift`